sdks/java/core/src/test/java/org/apache/beam/SdkCoreApiSurfaceTest.java

```diff
@@ -39,6 +39,7 @@
    */
   public static ApiSurface getSdkApiSurface(final ClassLoader classLoader) throws IOException {
     return ApiSurface.ofPackage("org.apache.beam", classLoader)
+        .pruningPrefix("org.apache.beam.repackaged")
```

Comment:
Add a JIRA + TODO about figuring out why we are exposing this. The investigation AND fix should be a release blocker.